### PR TITLE
Clean up and update readme

### PR DESCRIPTION
**Pull request by Thog**
This clean up and improve the README a bit.

Also remove CONFIG.md.

### CODE REVIEW

**Review by AcK77 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,15 +16,25 @@
 </p>
 
 <h5 align="center">
-    As of June 2020, Ryujinx goes past menus and in-game on over 1,000 commercial titles. Of those, roughly half are considered playable. See the compatiblity list <a href="https://github.com/Ryujinx/Ryujinx-Games-List/issues" target="_blank">here</a>.
+    As of August 2020, Ryujinx goes past menus and in-game on over 1,500 commercial titles. Of those, more than half are considered playable. See the compatiblity list <a href="https://github.com/Ryujinx/Ryujinx-Games-List/issues" target="_blank">here</a>.
```
Comment (EmulationFanatic) [inline README.md:19]:
```suggestion
    As of August 2020, Ryujinx goes past menus and in-game on over 1,500 commercial titles. Of those, more than half are considered playable. See the compatibility list <a href="https://github.com/Ryujinx/Ryujinx-Games-List/issues" target="_blank">here</a>.
```

File: README.md [hunk 2]
```diff
@@ -39,39 +49,15 @@ Download the ZIP Tarball. Then extract it to a directory of your choice.
 **Step three:**
 
 Build the App using a Command prompt in the project directory. You can quickly access it by holding shift in explorer (in the Ryujinx directory) then right clicking, and typing the following command:  
-Run `dotnet build -c Release -r win10-x64` inside the Ryujinx project folder to build Ryujinx binaries.
+Run `dotnet build -c Release -r win-x64` inside the Ryujinx project folder to build Ryujinx binaries.
 
 Every file related to Ryujinx is stored in the `Ryujinx` folder. This folder is located in the user folder, which can be accessed by clicking `Open Ryujinx Folder` under the File menu in the GUI.
```
Comment (EmulationFanatic) [inline README.md:54]:
```suggestion
Ryujinx system files are stored in the `Ryujinx` folder. This folder is located in the user folder, which can be accessed by clicking `Open Ryujinx Folder` under the File menu in the GUI.
```

File: README.md (hunk 1 above)
Comment (EmulationFanatic) [reply README.md:19]: typo/misspelling leftover from my own changes originally.

File: README.md (hunk 2 above)
Comment (EmulationFanatic) [reply README.md:54]: Clarifies that it is Ryujinx system files stored here. Not all Ryujinx files are stored there, as the program folder with the binaries still exists elsewhere.

File: README.md [hunk 3]
```diff
@@ -39,39 +49,15 @@ Download the ZIP Tarball. Then extract it to a directory of your choice.
...
-   Homebrew is available on many websites, such as the [Switch Appstore](https://www.switchbru.com/appstore/).
-
-   A hacked Nintendo Switch is needed to dump games, which you can learn how to do [here](https://nh-server.github.io/switch-guide/). Once you have hacked your Nintendo Switch, you will need to dump your own games with [NxDumpTool](https://github.com/DarkMatterCore/nxdumptool/releases) to get an XCI or NSP dump.
-
 ## Features
 
  - **Audio**
 
-   Audio is partially supported. We use C# wrappers for [OpenAL](https://openal.org/downloads/OpenAL11CoreSDK.zip) (installation needed), the main audio backend, and [libsoundio](http://libsound.io/) as the fallback. Our current Opus implementation is incomplete.
+   Audio output is entirely supported, audio input (microphone) isn't supported. We use C# wrappers for [OpenAL](https://openal.org/downloads/OpenAL11CoreSDK.zip) (installation needed), and [libsoundio](http://libsound.io/) as the fallback.
```
Comment (EmulationFanatic) [inline README.md:60]:
```suggestion
   Audio output is entirely supported, audio input (microphone) isn't supported. We use C# wrappers for [OpenAL](https://openal.org/downloads/oalinst.zip) (installation needed), and [libsoundio](http://libsound.io/) as the fallback.
```

Comment (EmulationFanatic) [reply README.md:60]: We want them to install the openAL windows binary, not the core SDK. Not sure why the last version had the wrong link too.

File: README.md [hunk 4]
```diff
@@ -107,9 +91,9 @@ Don't hesitate to open a new issue if a game isn't already on there.
 
 ## Help
 
-If you have homebrew or a particular game marked playable or in-game in our compatibility list that doesn't work within the emulator, you can contact us through our Discord. We'll take note of whatever is causing the app/game to not work, put it on the watch list and fix it at a later date.
+If you have homebrew or a particular game marked playable or in-game in our compatibility list that doesn't work within the emulator, you can contact us through our [Discord server](https://discord.gg/N2FmfVc). We'll take note of whatever is causing the app/game to not work, put it on the watch list and fix it at a later date.
```
Comment (EmulationFanatic) [inline README.md:94]:
```suggestion
If you are having problems launching homebrew or a particular game marked status-playable or status-ingame in our compatibility list, you can contact us through our [Discord server](https://discord.gg/N2FmfVc). We'll take note of whatever is causing the app/game to not work, put it on the watch list and fix it at a later date.
```

Comment (EmulationFanatic) [reply README.md:94]: General rewording for clarification/accuracy.

File: Ryujinx.HLE/HOS/Services/Time/TimeZone/TimeZoneContentManager.cs [hunk 5]
```diff
@@ -22,7 +22,7 @@ public class TimeZoneContentManager
     {
         private const long TimeZoneBinaryTitleId = 0x010000000000080E;
 
-        private readonly string TimeZoneSystemTitleMissingErrorMessage = "TimeZoneBinary system title not found! TimeZone conversions will not work, provide the system archive to fix this error. (See https://github.com/Ryujinx/Ryujinx#requirements for more information)";
+        private readonly string TimeZoneSystemTitleMissingErrorMessage = "TimeZoneBinary system title not found! TimeZone conversions will not work, provide the system archive to fix this error. (See https://github.com/Ryujinx/Ryujinx/wiki/Ryujinx-Setup-&-Configuration-Guide#initial-setup-continued---installation-of-firmware for more informations)";
```
Comment (EmulationFanatic) [inline Ryujinx.HLE/HOS/Services/Time/TimeZone/TimeZoneContentManager.cs:25]:
```suggestion
        private readonly string TimeZoneSystemTitleMissingErrorMessage = "TimeZoneBinary system title not found! TimeZone conversions will not work, provide the system archive to fix this error. (See https://github.com/Ryujinx/Ryujinx/wiki/Ryujinx-Setup-&-Configuration-Guide#initial-setup-continued---installation-of-firmware for more information)";
```

Comment (EmulationFanatic) [reply Ryujinx.HLE/HOS/Services/Time/TimeZone/TimeZoneContentManager.cs:25]: nit

**Pull request merged**